web_src/js/features/repo-legacy.js

```diff
@@ -324,7 +324,7 @@ async function onEditContent(event) {
   const renderContent = segment.querySelector('.render-content');
   const rawContent = segment.querySelector('.raw-content');
 
-  let comboMarkdownEditor;
+  let comboMarkdownEditor, dropzoneInst;
```

Comment:
What if there are multiple dropzones on the same page? Single shared "global" instance seems bad for that.

Reply:
It is scoped, the same as `comboMarkdownEditor`

Reply:
I will rewrite this to make it clearer.

Reply:
786ed861c845fd457a4045f51da2a190fa11a127